src/stdatamodels/jwst/transforms/resources/schemas/stsci.edu/jwst_pipeline/miri_wfss_dispersion-1.0.0.yaml

```diff
@@ -0,0 +1,48 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/jwst_pipeline/miri_wfss_dispersion-1.0.0"
+title: >
+  MIRI WFSS dispersion model
+description: |
+  Supports one model:
+   - Given x,y,wave,order in effective direct image return x,y,wave,order in dispersed
+   - Given x,y,x0,y0,order in dispersed image returns x, y, wave, order in effective direct.
+anyOf:
+  - $ref: "http://stsci.edu/schemas/asdf/transform/transform-1.4.0"
+  - type: object
+    properties:
+      xmodels:
+        description: |
+          MIRI WFSS row dispersion model
+        type: array
+        items:
+          $ref: "http://stsci.edu/schemas/asdf/transform/transform-1.4.0"
```

Comment:
See my comment above but I think this does not match the reference file, where `xdisp` is a list of a list of 2 models.

Reply:
We have changed what xmodels is - it is now a 1D model